Yoast/wordpress-seo
# Remove the old snippet preview

## Summary

This PR can be summarized in the following changelog entry:

* Changes the snippet editor to be completely rendered by React. This will give us the possibility to add new features to it and use it on different pages.

## Relevant technical choices:

This will make the React Snippet Editor the one and only Snippet Editor.

* Because the App doesn't take the data from its internal snippet preview object anymore we need to pass this data in the data collectors.

## Test instructions

This PR can be tested by following these steps:

* Test in combination with https://github.com/Yoast/yoast-components/pull/537/files.
* Make sure there are no errors in the console. Please check posts, pages, tags and categories.
* When you put replacevars in the title or description, the progress bars and assessments should take the title with the replaced replacevars. 
* When something is changed in the title and description, the assessments should refresh.

## Quality assurance
